test/TritonGPU/reduce-data-duplication.mlir

```diff
@@ -12,3 +12,31 @@ module attributes {"triton_gpu.target" = "cuda:80", "triton_gpu.num-ctas" = 1 :
     tt.return
   }
 }
+
+// -----
+
+//       CHECK:   conversion_shortcut_blocked_dotop_warp32
```

Comment:
Nit: `CHECK-LABEL` is typically preferred for marking the boundaries.

Similary for other tests.

Reply:
Original test in the file do not use CHECK-LABEL so I decided to use same style here.

P.s. First test can not use CHECK-LABEL because it has regexp matched that is used after label

Reply:
You can use `$`-prefix to make captures persist across `CHECK-LABEL`: 

https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-FileCheck-enable-var-scope
> Variables with names that start with $ are considered global and remain set throughout the file.